lib/routes/universities/wzu/news.js

```diff
@@ -1,24 +1,80 @@
 const got = require('@/utils/got');
 const cheerio = require('cheerio');
+const { URL } = require('url');
 const { parseDate } = require('@/utils/parse-date');
 
+/* 新闻列表
+温大新闻 http://www.wzu.edu.cn/index/wdxw.htm
+媒体温大 http://www.wzu.edu.cn/index/mtwd.htm
+学术温大 http://www.wzu.edu.cn/index/xswd.htm
+通知公告 http://www.wzu.edu.cn/index/tzgg.htm
+招标信息 http://www.wzu.edu.cn/index/zbxx.htm
+学术公告 http://www.wzu.edu.cn/index/xsgg.htm
+*/
+
+const baseUrl = 'http://www.wzu.edu.cn/index/';
+
+const newsType = {
+    wdxw: '温大新闻',
+    mtwd: '媒体温大',
+    xswd: '学术温大',
+    tzgg: '通知公告',
+    zbxx: '招标信息',
+    xsgg: '学术公告',
+};
+
+/**
+ * @description: 抓取文章内容，本函数参考并修改自 whu/news.js，感谢原作者。
+ * @param {*} link
+ * @return {*} description
+ */
+async function load(link) {
+    // 请求文章页面
+    const newsResp = await got.get(link);
+    // 加载文章内容
+    const $ = cheerio.load(newsResp.data);
```

Comment:
```suggestion
    const $ = cheerio.load(newsResp.data, { decodeEntities: false });
```

解决文章中以 UTF8 编码显示中文字符